src/bluesky/bundlers.py

```diff
@@ -532,7 +523,7 @@ async def save(self, msg):
         # we do not have the descriptor cached, make it
         if descriptor_doc is None or d_objs is None:
             for obj in objs_read:
-                await self._ensure_cached(obj, collect=isinstance(obj, Collectable))
+                await self._ensure_cached(obj, collect=isinstance(obj, Collectable), stream_name=desc_key)
```

Comment:
Just spotted a historical bug here. `save` should always use the output of `describe()`, not `describe_collect()`, therefore this should be:
```suggestion
                await self._ensure_cached(obj, collect=False)
```